web/src/routes/SearchPage.tsx

```diff
@@ -25,6 +25,13 @@ const List = styled.ul`
   }
 `
 
+const SearchCounter = styled.p`
+  margin: 10px 0;
+  padding-left: 5px;
```

Comment:
As Integreat also supports languages that are right to left, like arabic, just adding padding left, will only work properly for left-to-right-languages. So you also need to add this padding right.